### ISSUE TITLE
Update 'Styling your course' wiki page

### ISSUE THREAD

**Issue by guywillis** [adaptlearning/adapt_framework#3494]
### Subject of the issue
The current 'Styling your course' wiki page is very outdated and refers to theme elements that have been removed. Page requires an extensive update to be relevant to the current state of the Vanilla theme.

**Comment by oliverfoster** [comment]
Can this be closed? @guywillis

**Comment by guywillis** [comment]
@oliverfoster No, the page in question still needs an update - https://github.com/adaptlearning/adapt_framework/wiki/Styling-your-course
